pkg/web/templates/components/verify_email.html

```diff
@@ -0,0 +1,21 @@
+{{ define "verify_email" }}
+
+<div id="verify-email" class="mx-auto max-w-4xl">
+  <p class="py-4 text-lg">
+    {Originator VASP Name} is requesting confirmation of Travel Rule data for a pending transaction. To securely
+    review the data and complete the transaction, please confirm your identity by entering your email address below.
+  </p>
+  <!-- TODO: Add htmx attributes -->
+  <form id="verify-email-form" class="py-2" method="post" hx-post="" hx-target="#verify-email" hx-swap="outerHTML" hx-ext="json-enc" hx-indicator="loader">
```

Comment:
After a user submits the form, a successful response will result in this div being replaced by the div in `/sunrise/verification.html`.